### DISCUSSION TITLE
Welcome to yiimp_install_script Discussions!

### DISCUSSION THREAD

**Discussion by afiniel** [afiniel/yiimp_install_script#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nymbuzcumulus** [comment]
Hello, Years ago Itryed to install Yiimp. Unfortunately got several bugs on those days and today I found it again ! 
I am willing to run the full concept since I really Like it since then. 

Now. Ubuntu has mopved up to 24.4 LTS ... I believe it should run as well. 
But If you think I should only run on 20.04, just let know.. all your advice will be good, and help will be very welcome! 
Thanks in advance for all this effort !

**Comment by nymbuzcumulus** [comment]
I have on another pc the whole Bitcoin Core Installed, My approach is to create the whole pool for a Solo miner stage, I am doing this cause I have several Asics and need them to properly find Blocks.. Whats your approach about Yiimp  and working this way ?

**Comment by finmine** [comment]
Hello,
Can you please add some missing algos like kawpow and firopow?
Thank you
